editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3705,6 +3728,17 @@ void CanvasItemEditor::_button_tool_select(int p_index) {
 	tool = (Tool)p_index;
 }
 
+void CanvasItemEditor::_update_override_camera_button(bool p_game_running) {
+	if (p_game_running) {
+		override_camera_button->set_disabled(false);
+		override_camera_button->set_tooltip(TTR(TOOLTIP_CAMERA_OVERRIDE_OK));
+	} else {
+		override_camera_button->set_disabled(true);
+		override_camera_button->set_pressed(false);
+		override_camera_button->set_tooltip(TTR(TOOLTIP_CAMERA_OVERRIDE_NO_GAME));
```

Comment:
This is not going to work, TTR is parsed, so it needs a direct string put in there